geomesa-quickstart-hbase/README.md

```diff
@@ -0,0 +1,236 @@
+ A8P                             00\x00@Observation.45\xB2Clemen\xF3\x01\x00\x00\x00\x00\x00\x00\x01\xC4\x01\x00\x00\x01CM8\x0
+                                 E\xA0\x01\x01\xC0S!\x93\xBCSg\x00\xC0CG\xBF$\x0DO\x7F\x80\x14\x1B$-?                         
+ \x08\xF8\x06\x03\x19\xDFf\xA3p\ column=D:\x06\x03\x19\xDFf\xA3p\x0CObservation.362, timestamp=1463593805680, value=\x02\x00\x
+ x0C                             00\x00@Observation.36\xB2Clemen\xF3\x01\x00\x00\x00\x00\x00\x00\x01j\x01\x00\x00\x01CQ\x17wh\
+                                 x01\x01\xC0S\x05\xA5b\xD49"\xC0B\x88*~\xD1\xA0}\x80\x14\x1B$-?                               
+ \x08\xF8\x06\x07\x19S\xD0\xA21> column=D:\x06\x07\x19S\xD0\xA21>Observation.35, timestamp=1463593805664, value=\x02\x00\x00\x
+                                 00?Observation.3\xB5Clemen\xF3\x01\x00\x00\x00\x00\x00\x00\x00#\x01\x00\x00\x01CS?`x\x01\x01\
+                                 xC0S_\xA7+G\xADH\xC0B\x90\xEB\xF7`\xC2T\x80\x13\x1A#,> 
+```
+
+## Visualize the data with GeoServer (optional)
+
+### Setting up the GeoMesa HBase GeoServer plugin
+
+Clone the GeoMesa source distribution:
+
+```bash
+$ git clone git@github.com:locationtech/geomesa.git
+$ cd geomesa
+```
+
+and build it:
+
+```
+$ mvn clean install -DskipTests
+```
+
+Go into the buld HBase GeoServer plugin module:
+
+```
+$ cd geomesa-gs-plugin/geomesa-hbase-gs-plugin/target
+```
+
+and extract the contents of the ``geomesa-hbase-gs-plugin-<version>-install.tar.gz``
+file into GeoServer's ``WEB-INF/lib`` directory. This distribution does not include
+the Hadoop or Zookeeper JARs; the following JARs should be copied from the ``lib``
+directory of your HBase or Hadoop installations into GeoServer's ``WEB-INF/lib``
+directory:
+
+ * hadoop-annotations-2.5.1.jar
+ * hadoop-auth-2.5.1.jar
+ * hadoop-common-2.5.1.jar
+ * hadoop-mapreduce-client-core-2.5.1.jar
+ * hadoop-yarn-api-2.5.1.jar
+ * hadoop-yarn-common-2.5.1.jar
+ * zookeeper-3.4.6.jar
+ * commons-configuration-1.6.jar
+
+(Note the versions may vary depending on your installation.)
+
+The HBase data store requires the configuration file `hbase-site.xml` to be on the
+classpath. This can be accomplished, for example in Tomcat, by placing the file in
+`WEB-INF/classes` (you should create the directory if it doesn't exist). Restart
+GeoServer.
+
+### Register the GeoMesa store with GeoServer
+
+Log into GeoServer using your user and password credentials. Click
+"Stores" and "Add new Store".  Select the ``HBase (GeoMesa)`` vector data source,
+and enter the following parameters.
+
+ * ``workspace`` this is dependent upon your GeoServer installation
+ * ``data source name`` pick a sensible name, such as, ``geomesa_quick_start``
+ * ``description`` this is strictly decorative; ``GeoMesa quick start``
+ * ``bigtable_table_name`` the name of the table you specified on the command line
```

Comment:
I remember the discussion from yesterday about "bigtable" still being a thing, but didn't realize it was so prominent.  That's a shame.